themes/digital.gov/src/scss/new/_collection.scss

```diff
@@ -30,6 +28,17 @@
     width: 0.9em;
```

Comment:
Can we use [spacing units](https://designsystem.digital.gov/design-tokens/spacing-units/) and deisgn tokens wherever possible?

I understand some won't be able to use and that's okay, but we should try to use.

Reply:
Got it, I've updated to use `units` where applicable.

Reply:
This is fine, but we should use `units()` for the `margin-left` and `width` in the future.